AnthropicEvaluator.py

```diff
@@ -0,0 +1,68 @@
+import os
+import tiktoken
+from LLMNeedleHaystackTester import LLMNeedleHaystackTester
+from anthropic import AsyncAnthropic, Anthropic
+
+
+class AnthropicEvaluator(LLMNeedleHaystackTester):
+    def __init__(self,**kwargs):
+        if 'anthropic_api_key' not in  kwargs and not os.getenv('ANTHROPIC_API_KEY'):
+            raise ValueError("Either anthropic_api_key must be supplied with init, or ANTHROPIC_API_KEY must be in env")
+
+        if 'model_name' not in kwargs:
+            raise ValueError("model_name must be supplied with init")
+        elif "claude" not in kwargs['model_name']:
```

Comment:
Consider using `"` or `'`. We may also consider using [Black](https://github.com/psf/black) to fix those inconsistencies automatically.

Reply:
fixed the inconsistencies.

<img width="657" alt="image" src="https://github.com/gkamradt/LLMTest_NeedleInAHaystack/assets/3776681/081452a1-4f28-47a1-add9-7af4b381b19f">